bin/telemetry_sim_ATHS

```diff
@@ -56,6 +56,15 @@ def cmdline():
     parser.add_argument("--focusz", action="store", default=13, type=float,
                         help="ATHexapod/ATAOS FocusZ position (mm)")
 
+    parser.add_argument("--instport", action="store", default=1, type=int,
+                        help="The instrument port where LATISS is installed")
+    parser.add_argument("--m3port", action="store", default=1, type=int,
+                        help="Port the M3 is directing light into")
+    parser.add_argument("--tracksys", action="store", default="Local", type=str,
+                        help="The track system value will [Local, Sidereal, or non-sidereal.]")
```

Comment:
The tracking coordinate system [Local, Sidereal, or non-sidereal].

Reply:
Thanks, fixed.